swift/ql/lib/codeql/swift/security/SensitiveExprs.qll

```diff
@@ -49,6 +49,7 @@ class SensitiveCredential extends SensitiveDataType, TCredential {
     exists(SensitiveDataClassification classification |
       not classification = SensitiveDataClassification::password() and // covered by `SensitivePassword`
       not classification = SensitiveDataClassification::id() and // not accurate enough
+      not classification = SensitiveDataClassification::private() and // covered by `SensitivePrivateInfo`
```

Comment:
I see you've excluded the new results from Swift for now.  Since they're a near duplicate of Swift's existing `class SensitivePrivateInfo` I think we should aim (now or later) to define that in terms of the new result classification instead, similarly to how `class SensitivePassword` already works.  This may add a few new results (for SSN and CCN) in the four or five Swift sensitive data queries.

Reply:
... I don't mind doing this as follow-up actually.  I already made half the changes I'll need to run a test of the SSN and CCN terms!

Reply:
PR for Swift: https://github.com/github/codeql/pull/16570